infra/terraform/modules/data_backup/data.tf

```diff
@@ -0,0 +1,51 @@
+data "aws_region" "current" {
+  current = true
+}
+
+data "aws_iam_policy_document" "assume_role_policy" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+    principals {
+      type        = "Service"
+      identifiers = ["ec2.amazonaws.com"]
+    }
+  }
+
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+    principals {
+      type        = "AWS"
+      identifiers = ["${var.k8s_worker_role_arn}"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "nfs_backup" {
+  statement {
+    effect = "Allow"
+    actions = [
```

Comment:
This looks good. I still need to have a look at the other PR, just saw it's using `s3cmd`, I don't know exactly what kind of operations will try to perform.

I was wondering if:
 * Will it need to perform other multipart-related actions?
 * Does it need to operate on versions?

It's good to start by keeping the policies to the minimum as in this case.

Reply:
So far in my testing it hasn't needed any other permissions - other multipartupload actions would be required for large file uploads, but >100MB uploads have worked successfully - similarly s3cmd doesn't interact with versions directly and just uploads/deletes files, with the bucket itself taking care of versioning.